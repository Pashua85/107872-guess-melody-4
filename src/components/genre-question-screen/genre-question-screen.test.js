import React from 'react';
import renderer from 'react-test-renderer';
import GenreQuestionScreen from './genre-question-screen';

describe(`GenreQuestionScreen`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(<GenreQuestionScreen/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
