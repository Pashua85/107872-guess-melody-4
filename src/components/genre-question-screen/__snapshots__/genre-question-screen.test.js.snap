// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreQuestionScreen should render correctly 1`] = `ShallowWrapper {}`;
